cfme/tests/infrastructure/test_webmks_console.py

```diff
@@ -80,6 +80,9 @@ def test_webmks_vm_console(request, appliance, provider, vm_obj,
     request.addfinalizer(vm_console.close_console_window)
     request.addfinalizer(appliance.server.logout)
     try:
+        if appliance.version > '5.9':
```

Comment:
use `appliance.version >= '5.9'` or `appliance.version > '5.8'` instead

Reply:
It should be **REQUIRED**